frontend/package.json

```diff
@@ -100,7 +100,7 @@
     "eslint-plugin-import": "2.25.4",
     "eslint-plugin-prettier": "4.0.0",
     "eslint-plugin-react": "7.28.0",
-    "jest-canvas-mock": "2.3.1",
+    "jest-canvas-mock": "2.5.2",
```

Comment:
Need to upgrade this or I'll encounter [this problem](https://stackoverflow.com/questions/75414699/jest-typeerror-cannot-redefine-property-window) when running frontend tests locally.